src/core/vectortile/qgsvectortileutils.h

```diff
@@ -48,6 +51,12 @@ class CORE_EXPORT QgsVectorTileUtils
 {
   public:
 
+    /**
+     * Parses the style URL to update the source URLs in the \a uri.
+     * \since QGIS 3.40
+     */
+    static bool updateUriSources( QString &uri SIP_OUT );
```

Comment:
I think we should either transform this function to a void, or add some explanation in the documentation as to why there's a bool return here.